databricks/sdk/retries.py

```diff
@@ -0,0 +1,53 @@
+import functools
+import logging
+import time
+from datetime import timedelta
+from random import random
+from typing import Callable, List, Optional, Type
+
+logger = logging.getLogger('databricks.sdk')
+
+
+def retried(*,
+            on: List[Type[BaseException]] = None,
```

Comment:
On is not used except in tests. Better to remove it here and add a callable helper that can implement the same.

Reply:
future code will use it